EasyPost/Address.cs

```diff
@@ -92,8 +87,8 @@ public static async Task<AddressCollection> All(Dictionary<string, object>? para
         ///     * {"country", string}
         ///     * {"phone", string}
         ///     * {"email", string}
-        ///     * {"verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
-        ///     * {"strict_verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
+        ///     * {"verifications", object} Possible items are true, "delivery" and "zip4".
+        ///     * {"strict_verifications", object} Possible items are true, "delivery" and "zip4".
```

Comment:
Question: why object? It should be one of list, bool, str. Now people will think it must be an object of some undefined type?

Reply:
There's no such thing as "one of multiple types" in a type-strict language 😉 The way to do that is say it's the base type `object`

Reply:
Okie